ci/pipeline

```diff
@@ -279,11 +274,12 @@ def release(requestVersion: String, gitSha: String, latest: Boolean = false): Un
     println("Docker image mesosphere/marathon:latest NOT updated")
   }
 
+  %('sbt, "publish")
```

Comment:
as @timcharper said in the other review, it would be nice to put a comment here...